### PR TITLE
[DFMD-4] Add Page Navigation to your Pages

### PR DESCRIPTION
**Pull request by MoyinoluwaA**

### CODE REVIEW

**Comment by MoyinoluwaA** [comment]
Screenshot (72)

**Comment by MoyinoluwaA** [comment]
Screenshot (73)

File: submissions/moyinoluwaAdenuga/HTML/index.html [hunk 1]
```diff
@@ -12,7 +12,7 @@
             <li><a href="#">Home</a></li>
             <li><a href="#">About</a></li>
             <li><a href="#">Collection</a></li>
-            <li><a href="#">Contact</a></li>
+            <li><a href="../html-forms/contactform.html">Contact Form</a></li>
```
Comment (Ireti203) [inline submissions/moyinoluwaAdenuga/HTML/index.html:15]: You should also have the link to index.html

**Review by Ireti203 — CHANGES_REQUESTED** [review]

**Comment by MoyinoluwaA** [comment]
Screenshot (74)

**Review by adaugochi — APPROVED** [review]

**Pull request merged**